provisioning/miniprov/create-template.sh

```diff
@@ -23,9 +23,9 @@ FILENAME=$1
 URL="json://$(readlink -f "$FILENAME")"
 
 conf "$URL" set 'cluster>my-cluster>site>storage_set_count=1'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>data=1'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>parity=0'
-conf "$URL" set 'cluster>my-cluster>storage_set[0]>spare=0'
+conf "$URL" set 'cluster>my-cluster>storage_set[0]>sns>data=1'
```

Comment:
According to below doc, actual path is '_cluster>{cluster-id}>storage_set[N]>{type}>durability>data_'.
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys.

@knekrasov I guess you assuming that this will be handled by https://github.com/Seagate/cortx-hare/pull/1576/files. Is is correct?

Reply:
Yes, I would prefer to keep the changes local. If your patch addresses missing "durability" key AND if my patch is landed before yours, then your patch should be rebased and updated in this place also.